core/src/main/java/com/equo/application/client/EquoApp.java

```diff
@@ -25,8 +25,8 @@ public class EquoApp {
   public static final String NEW_TAB_URL_SWITCH = "--new-tab-url";
   public static final String APP_ID_SWITCH = "--app-id";
   private static final String CLASSPATH_SCHEME = "equo";
-  private static final String CUSTOM_URL = "equo.app";
-  private static final String CLASSPATH_URI =
+  private static String CUSTOM_URL = "equo.app";
```

Comment:
change to `null`

Reply:
No, there is a possibility to call create() without specifying a name, in which case "equo.app" will be the default value